docs/source/collect/goldpinger.md

```diff
@@ -5,13 +5,18 @@ description: Gathering goldpinger pod checks from a kubernetes cluster
 
 The `goldpinger` collector is used to collect pod-ping checks gathered by a [goldpinger service](https://github.com/bloomberg/goldpinger) installed in a kubernetes cluster. The collector makes a request to `<host>/check_all` endpoint. Periodically, this service will ping pods running on every node (daemonset pods) to ensure that nodes can reach all other nodes in that cluster. It caches these information and surfaces it via http endpoints. If this collector is run within a kubernetes cluster, the collector will directly make the http request to the goldpoinger endpoint (`http://goldpinger.<namespace>.svc.cluster.local:80/check_all`). If not, the collector attempts to launch a pod in the cluster and makes the request within the running container.
 
+If goldpinger is not installed, the collector will attempt to temporarily install it until the collector has completed.
```

Comment:
I know `temporarily install` means we remove after, but would it be better to explicitly say we remove goldpinger?